requirements.txt

```diff
@@ -7,8 +7,6 @@ tox==3.14.0
 coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
-
-setuptools
```

Comment:
We do need setuptools for local development
Add it back to requirements.txt

Reply:
Damn, I didnt even see that